### PR TITLE
Completed asssignment 6

### PR DESCRIPTION
**Pull request by dnyaneshwarkhule**
complete assignment 6 check

### CODE REVIEW

File: dnyaneshwar.htmlfile/from.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/from.css:1]: Remove this extra line only

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+     <h1> MI-9 Intern HTML ASSESMENT </h1>
+      </div> <br>
+      <div> <hr>
+       </div>
+
+<div>
+      <h1>personal details</i> </h1> </div>
+
+       <form>
+    <label for="Fname"><b> First name:</b> </label>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Make Proper Indentation

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+<input type="text" id="fName" name="fName"> <br> <br>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same Here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+<label for="LName">  <b>LName: </b></label>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    <input type="checkbox" value="18-25"  checked>
+    <label for="25-30"> 26-40</label>
+    <input type="checkbox"value ="26-40">
+    <label for="15-18"> 15-18</label> 
+    <input type="checkbox" value="15-18"> <br> <br> <br>
+    <div>
+
+   <h1> acadmic details </h1> </div>
+
+<label for="skillset"> <b>skillset: </b></label>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+<label for="html">html </label>
+<input type="checkbox" value="html">
+<label for="javascript"> javascript</label>
+<input type="checkbox" value="javascript">
+<label for="java"> java</label>
+<input type="checkbox" value="java">
+<label for="oracle"> oracle</label>
+<input type="checkbox" value="oracle">
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: same here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    <label for="25-30"> 26-40</label>
+    <input type="checkbox"value ="26-40">
+    <label for="15-18"> 15-18</label> 
+    <input type="checkbox" value="15-18"> <br> <br> <br>
+    <div>
+
+   <h1> acadmic details </h1> </div>
+
+<label for="skillset"> <b>skillset: </b></label>
+<label for="html">html </label>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same Here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+<input type="checkbox" value="html">
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

File: dnyaneshwar.htmlfile/fromtagex.html (hunk 6 above)
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+<label for="javascript"> javascript</label>
+<input type="checkbox" value="javascript">
+<label for="java"> java</label>
+<input type="checkbox" value="java">
+<label for="oracle"> oracle</label>
+<input type="checkbox" value="oracle">
+
+</form>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

File: dnyaneshwar.htmlfile/fromtagex.html (hunk 6 above)
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here add indentation

Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here

File: dnyaneshwar.htmlfile/fromtagex.html [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    <label for="15-18"> 15-18</label> 
+    <input type="checkbox" value="15-18"> <br> <br> <br>
+    <div>
+
+   <h1> acadmic details </h1> </div>
+
+<label for="skillset"> <b>skillset: </b></label>
+<label for="html">html </label>
+<input type="checkbox" value="html">
+<label for="javascript"> javascript</label>
```
Comment (MKadeer17) [inline dnyaneshwar.htmlfile/fromtagex.html]: Same here to last form tag

**Review by MKadeer17 — COMMENTED** [review]
Add indentation only remove extra line in css

File: dnyaneshwar.htmlfile/fromtagex.html (hunk 2 above)
Comment (priya-ughade) [reply dnyaneshwar.htmlfile/fromtagex.html]: Yes . Remove extra space

**Review by MKadeer17 — APPROVED** [review]

**Pull request merged**